content/strategic/wage-war/plugin.js

```diff
@@ -80,7 +80,7 @@ const calcWarringPotential = (df) => (predator, defenseThreshold = 0) => (prey)
     }
     try {
         const sending = predator.energyCap * (parseInt(100 - defenseThreshold) / 100); // @todo: calculate predator energyCap as a factor of full energyCap at max level instead of predator.energyCap.
-        const arriving = Math.floor(df.getEnergyArrivingForMove(predator.locationId, prey.locationId, sending) / (prey.defense / 100)); // @todo: calcualte prey defense as factor of max-level defense possible.
+        const arriving = Math.floor(df.getEnergyArrivingForMove(predator.locationId, prey.locationId, df.getDist(predator.locationId, prey.locationId), sending) / (prey.defense / 100)); // @todo: calcualte prey defense as factor of max-level defense possible.
```

Comment:
I believe you just want this to be `undefined` instead of trying to calc the distance. Putting it as undefined uses the old logic.